srcpkgs/libnma/template

```diff
@@ -14,7 +14,7 @@ license="GPL-2.0-only, LGPL-2.1-or-later"
 homepage="https://gitlab.gnome.org/GNOME/libnma"
 changelog="https://gitlab.gnome.org/GNOME/libnma/-/raw/master/NEWS"
```

Comment:
```suggestion
changelog="https://gitlab.gnome.org/GNOME/libnma/-/raw/main/NEWS"
```
according to the changelog, this release just added experimental gtk4 support which we don't currently enable

Reply:
The gtk4 library is currently disabled by default. Would you like me to explicitly disable it?